bool clear_waypoints
---
bool success